### PR TITLE
[VUFIND-1603] Add unit tests to cover SearchTabs and SearchTabsHelper.

### PR DESCRIPTION
**Pull request by EreMaijala**
This brings unit test coverage of SearchTabs view helper and SearchTabsHelper to 100%.

### CODE REVIEW

File: module/VuFind/tests/unit-tests/src/VuFindTest/View/Helper/Root/SearchTabsTest.php [hunk 1]
```diff
@@ -162,15 +283,50 @@ protected function getHelper(
...
+                    $request,
+                    [
+                        'Solr:filtered' => 'logged-in',
+                    ],
+                    [],
+                ]
+            )->onlyMethods(['getHiddenFilters', 'getTabConfig', 'getTabFilterConfig'])
             ->getMock();
-        $searchTabsHelper->expects($this->exactly($filtersCalls))
+        /*$searchTabsHelper->expects($this->exactly($filtersCalls))
```
Comment (demiankatz) [inline module/VuFind/tests/unit-tests/src/VuFindTest/View/Helper/Root/SearchTabsTest.php]: Is this commented-out code left here for a reason?

**Review by demiankatz — CHANGES_REQUESTED** [review]
Thanks, @EreMaijala -- just one small question about possible cleanup.

File: module/VuFind/tests/unit-tests/src/VuFindTest/View/Helper/Root/SearchTabsTest.php (hunk 1 above)
Comment (EreMaijala) [reply module/VuFind/tests/unit-tests/src/VuFindTest/View/Helper/Root/SearchTabsTest.php]: Oops, no. Now cleaned up!

**Review by demiankatz — APPROVED** [review]
Thanks, @EreMaijala!

**Pull request merged**